hsakmt-roct/PKGBUILD

```diff
@@ -1,62 +1,34 @@
-# Maintainer: Olaf Leidinger <oleid@mescharet.de>
+# Maintainer: acxz <akashpatel2008 at yahoo dot com>
+# Contributor: Jakub Okoński <jakub@okonski.org>
+# Contributor: Olaf Leidinger <oleid@mescharet.de>
+# Contributor: Ranieri Althoff <ranisalt+aur at gmail.com>
+
 pkgname=hsakmt-roct
-pkgver=1.8.0.r0.e3dd067
-pkgrel=2
+pkgver=3.1.0
+pkgrel=1
 pkgdesc="Radeon Open Compute Thunk Interface"
-_gitdir=ROCT-Thunk-Interface
 arch=('x86_64')
-url="https://github.com/RadeonOpenCompute/ROCT-Thunk-Interface"
-license=('X11/MIT')
-groups=()
-depends=(pciutils numactl)
-makedepends=(git cmake gcc) 
-provides=("${pkgname%-git}")
-conflicts=("${pkgname%-git}")
-replaces=()
-backup=()
-options=()
-source=('git+https://github.com/RadeonOpenCompute/ROCT-Thunk-Interface.git#branch=roc-1.8.x')
-md5sums=('SKIP')
-
-pkgver() {
-	cd "$srcdir/${_gitdir}"
-
-	# Git, tags available
-	printf "%s" "$(git describe --long | sed -e 's:roc-::g'  -e 's/\([^-]*-\)g/r\1/;s/-/./g')"
-}
+url='https://github.com/RadeonOpenCompute/ROCT-Thunk-Interface'
+license=('MIT')
+depends=('numactl' 'pciutils')
+makedepends=('cmake')
+provides=("roct-thunk-interface=$pkgver")
+conflicts=('roct-thunk-interface')
+replaces=('roct-thunk-interface')
+source=("$pkgname::git+$url#tag=roc-$pkgver")
+sha256sums=('SKIP')
```

Comment:
can you add a checksum here? this is not a git package.

Reply:
I understand this might work for github actions, but we should not move away from packaging standards to get CI working. We should bend CI to work with our packaging standards.

Reply:
It downloads from a git tag. A -git package would download the latest commit from master (omit `#tag=`)

Reply:
This then requires a git dep tho right? Let us stick with the release version instead of using git at all.